assets/stylesheets/_index.sass

```diff
@@ -37,6 +37,11 @@ a
   color: $text-color-dark
   padding: 50px
 
+  .sorting
```

Comment:
What about this?

```sass
.sorting
  padding-bottom: 50px
```

It keeps everything aligned on the left, but with a bit of white space between the list and the sorting actions.